src/ipelib/dynamo/dynamo_module.f

```diff
@@ -157,7 +157,8 @@ module dynamo_module
 !
 ! am 10/04 define potential model : NONE or HEELIS - first test NONE
 !       character(len=*),parameter :: potential_model='NONE'
-       character(len=*),parameter :: potential_model='HEELIS'
+!      character(len=*),parameter :: potential_model='HEELIS'
```

Comment:
This could be setup as input option if helpful.

Reply:
Yes, we might need to bring this out. Needs to be decided later